### DISCUSSION TITLE
Text in the windows installer is truncated

### DISCUSSION THREAD

**Discussion by arye-eidelman** [nodejs/help#4655] [Q&A]
* **Version**: node-v12.19.0-x64.msi
* **Platform**: Windows 10 64bit
* **Subsystem**:

### What steps will reproduce the bug?
Installing node on a new machine.

### What is the expected behavior?

If text in the installer is too long either add a show more button or link to a support page for more info.

There seems to be plenty more space though.

### What do you see instead?

Some text in the installer is truncated with an ellipsis. it's on the screen regarding installing c/c++ build tools and Chocolatey.
![image](https://user-images.githubusercontent.com/7709411/96906670-254eaf00-1468-11eb-8f98-80a8a82181bd.png)

**Comment by ridays2001** [comment]
There's another issue with the windows installer. If you minimize it while installing, then it won't open properly after installation, or when the window closes and reopens. See the screenshot.

![image](https://user-images.githubusercontent.com/46166258/97130050-5aa81500-1766-11eb-8e77-a87c20f5a990.png)

**Comment by zhaparoff** [comment]
Faced the issue @ridays2001 mentioned numerous times already. Its really annoying.
Not sure if it is an issue of the installer framework or related to some settings in the node.js installer exactly.
I think its worth creating a separate issue in the tracker.

Faced so far in:
OS: Windows 10, Windows Server 2016, Windows Server 2019 (all x64)
Node versions: 10, 11, 12, 13, 14

**Comment by avivkeller** [comment]
Hi! I converted this to a discussion, as it's possible this issue has to do with the framework behind the installer and not Node.js itself.

@nodejs/platform-windows 

(I don't work with the installer often, so what I am saying might be wrong).
